graphql/function.json

```diff
@@ -8,7 +8,8 @@
             "methods": [
                 "get",
                 "post"
-            ]
+            ],
+            "route": "{*path}"
```

Comment:
This, combined with `routePrefix: ""` in host.json, routes all requests to this function.